src/Spice86/Views/RegistersView.axaml

```diff
@@ -0,0 +1,163 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:viewModels="clr-namespace:Spice86.ViewModels"
+             xmlns:converters="clr-namespace:Spice86.Converters"
+             xmlns:behaviors="clr-namespace:Spice86.Behaviors"
+             mc:Ignorable="d" d:DesignWidth="250" d:DesignHeight="450"
+             x:Class="Spice86.Views.RegistersView"
+             x:DataType="viewModels:IRegistersViewModel">
+    <Grid RowDefinitions="Auto,Auto,Auto,Auto,Auto">
+        <!-- Header -->
+        <TextBlock Grid.Row="0"
+                   Text="CPU Registers"
+                   FontWeight="Bold"
+                   Margin="0,0,0,8"
+                   HorizontalAlignment="Center" />
+
+        <!-- General Purpose Registers -->
+        <Border Grid.Row="1"
+                BorderBrush="{DynamicResource SemiColorBorder}"
+                BorderThickness="1"
+                CornerRadius="4"
+                Padding="8"
+                Margin="0,0,0,8">
+            <StackPanel>
+                <ItemsControl ItemsSource="{Binding GeneralRegisters}">
+                    <ItemsControl.ItemTemplate>
+                        <DataTemplate>
+                            <Grid ColumnDefinitions="Auto,*" Margin="0,2">
+                                <TextBlock Grid.Column="0"
+                                           Text="{Binding Name}"
+                                           FontFamily="Consolas,Menlo,Monaco,monospace"
+                                           Width="40"
```

Comment:
MinWidth should be used instead of fixed Height/Width to allow for responsive design.

Reply:
Done.